handlers/api.go

```diff
@@ -251,7 +251,18 @@ func ApiEthStoreDay(w http.ResponseWriter, r *http.Request) {
 	}
 	defer rows.Close()
 
-	returnQueryResults(rows, w, r)
+	dayTime := func(dataEntryMap map[string]interface{}) error {
```

Comment:
This same `dayTime` function exists twice now but I didn't feel it was right to have it outside the functions they are in.
What do you think?

Reply:
I understand your decision.